akka-actor/src/main/scala/akka/actor/ActorRef.scala

```diff
@@ -103,7 +103,7 @@ object ActorRef {
  * the unique id of the actor is not taken into account when comparing actor paths.
  */
 abstract class ActorRef extends java.lang.Comparable[ActorRef] with Serializable {
-  scalaRef: InternalActorRef =>
+  scalaRef: InternalActorRef with ActorRefScope =>
```

Comment:
what is the reason here?

Reply:
Similar, with this:
```scala
class InternalActorRef extends ActorRef with ScalaActorRef {
    this: ActorRefScope =>
}
class ActorRef .. {
    scalaRef: InternalActorRef =>
}
```
=> need change to
```scala
    scalaRef: InternalActorRef with ActorRefScope =>
```